bcycle-backend/Models/Trip.cs

```diff
@@ -6,29 +6,38 @@
 
 namespace bcycle_backend.Models
 {
+    // TODO: can we remove those comments?
     public class Trip
     {
-        public int ID { get; set; }
+        public int Id { get; set; }
         public double Distance { get; set; }
         public int Time { get; set; }
         public DateTime Started { get; set; }
         public DateTime Finished { get; set; }
         public string MapImageUrl { get; set; }
-        public int UserID { get; set; }
-        public int? GroupTripID { get; set; }
+        public string UserId { get; set; }
```

Comment:
We also need to get rid of `User`, fix the types in other models, and generate new migrations. (We can just replace delete the current Migrations folder and generate a new `InitialCreate` at this stage, since that won’t lose any data, and because IIRC there are limitations with SQLite support because it doesn’t support `ALTER TABLE`)